tests/integration/feature_store_integration/test_feature_materialize.py

```diff
@@ -404,18 +421,24 @@ def test_online_features(config, deployed_feature_list):
             # Note: shouldn't have to provide this once parent entity lookup is supported (via child
             # entity üser id)
             "user_status": "STÀTUS_CODE_37",
+            "order_id": "T1230",
         }
     ]
     data = OnlineFeaturesRequestPayload(entity_serving_names=entity_serving_names)
 
     tic = time.time()
-    res = client.post(
-        f"/deployment/{deployment.id}/online_features",
-        json=data.json_dict(),
-    )
+    # Note: don't mock with freezegun since it also freezes elapsed time tracking for logging
+    # purpose within OnlineServingService
+    with patch("featurebyte.service.online_serving.datetime", autospec=True) as mock_datetime:
+        mock_datetime.utcnow.return_value = datetime(2001, 1, 2, 12)
+        res = client.post(
+            f"/deployment/{deployment.id}/online_features",
+            json=data.json_dict(),
+        )
```

Comment:
Another solution is to set `tick=True` in `freezegun` like `@freezegun.freeze_time("2001-01-02 12:00:00", tick=True)`

Reply:
I see, good to know! Though to avoid unintended effects due to time ticking slightly forward, will stick to manually patching here.